src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/RiskControllerTest.kt

```diff
@@ -16,13 +16,13 @@ class RiskControllerTest(
 ) : IntegrationTestBase() {
 
   @Test
-  fun `retrieves risk data when no mappa details available`() {
+  fun `retrieves personal details data when no MAPPA or RoSH details available`() {
```

Comment:
I can't see any assertion in this test that confirms the personal details are in the response?

Reply:
![image](https://user-images.githubusercontent.com/104494371/175533479-4f401523-6e6a-46c8-b8ed-62014631163f.png)
They are there, see the image :)

Reply:
👍